src/uiTest/resources/ideprobe.conf

```diff
@@ -2,6 +2,7 @@ probe {
   paths {
     base = ${?IDEPROBE_PATHS_BASE}
     screenshots = ${?IDEPROBE_PATHS_SCREENSHOTS}
+    diagnostics = "/root/.ideprobe-uitests/logs"
```

Comment:
Whoops this will only work in the CI, locally it's bound to fail (as there won't be access to `/root/`). Pls check `com/virtuslab/gitmachete/buildsrc/UiTests.kt:48` for how it can be done

Reply:
Sure, this line was just for quick testing on the. circle CI. I will set the directory programatically for this PR :-)